internals/router/routes.go

```diff
@@ -169,6 +169,8 @@ func engineRouter() http.Handler {
 
 	r.Get("/connector/{id}/executions/last", handlers.GetlastConnectorExecutionDateTime)
 
+	r.Get("/export/facts/{id}", handlers.ExportFact)
```

Comment:
Convention de nommage à discuter ?
```suggestion
	r.Get("/facts/{id}/export", handlers.ExportFact)
```

Reply:
c'est vrai je préfère ça aussi